src/cloudWatchLogs/document/logDataDocumentProvider.ts

```diff
@@ -29,12 +26,13 @@ export class LogDataDocumentProvider implements vscode.TextDocumentContentProvid
     }
 
     public provideTextDocumentContent(uri: vscode.Uri): string {
-        // get latest content and return
-        const content = this.registry.getLogContent(uri, { timestamps: true })
-        if (!content) {
-            getLogger().error(`No content found for URI: ${uri.path}`)
+        if (!isCwlUri(uri)) {
+            return ''
```

Comment:
This is a silent failure. Odds are this document provider would only be called with a non-CWL URI if there's a bug somewhere else. I would at least log something here if we don't want to raise an exception.

Reply:
An existing test required this, but I think it is better to throw an error. I've updated the test and this code to throw an error instead.